### ISSUE TITLE
No such function [inputFromArray] when magento2:generate:schema-add-column

### ISSUE THREAD

**Issue by redo-interactive** [astorm/pestle#524]
Hi there,

when adding new column to the schema I have this issue : 

Columns Name? (new_column) ()] brand_id
No such function [inputFromArray] imported for namespace [Pulsestorm\Magento2\Generate\Schemaaddcolumn]

it seems that the function isn't included into the scope because I have found it in
modules/pulsesstorm/magento2/cli/generate/menu/module.php
and 
library/all.php

**Comment by astorm** [comment]
Thanks for the bug report @redo-interactive, we've confirmed the behavior.  It looks like we failed to import this function into the module with something like 

    pestle_import('Pulsestorm\Magento2\Cli\Generate\Menu\inputFromArray');

Also, if you're curious, the actual function definition [is here](https://github.com/astorm/pestle/blob/master/modules/pulsestorm/magento2/cli/generate/menu/module.php#L28).

We'll get a fix out ASAP.  Also, it looks like you can still use the command with the non-interactive  invocation

    pestle.phar magento2:generate:schema-add-column app/code/Pulsestorm/Pestle/upgrade_scripts/schema/0.0.5.php  pulsestorm_pestle_thing sub_title varchar

to avoid this error. See the [command docs](https://pestle.readthedocs.io/en/latest/magento2-mvc/#generateschema-add-column) for more information.

**Comment by redo-interactive** [comment]
astorm thanks for your replay, I have managed to solve it with building from your source phar file, just needed to include missing function in good namespace and exclude all.php from autoloader :).

**Comment by astorm** [comment]
@redo-interactive fixed in https://github.com/astorm/pestle/pull/525, and just released pestle 1.5.3 with this fix it in.  If you `pestle.phar selfupdate` you should get the latest version.

I'll be closing this issue out, but if you need more help please let me know.  Also -- I'd be curious what you mean by "excluding all.php from the autoloader" ... are you using pestle in a non-pestle.phar context?

**Comment by redo-interactive** [comment]
well as pestle.phar is binary, I have downloaded the source from your git repository, and compiled it with build.sh. After executing new phar file to create new column, there were errors saying that functions already were declared in file all.php, so I started to add conditions if function_exists in files that are located in modules/ but as it was too much to do I tried to exclude all.php from autoload.php and it worked. Sorry if I'm not explaining clearly :)... This is great tool that you have created! :)
